jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -231,3 +231,51 @@ def test_aperture_markers(cubeviz_helper, spectrum1d_cube):
         extract_plg._obj.vue_adopt_slice_as_reference()
         extract_plg._obj.vue_goto_reference_wavelength()
         assert slice_plg.slice == 1
+
+
+def test_cone_aperture(cubeviz_helper, spectrum1d_cube_larger):
+    from specutils import Spectrum1D
+    cubeviz_helper.load_data(spectrum1d_cube_larger)
+    cubeviz_helper.load_regions([CirclePixelRegion(PixCoord(1, 1), radius=0.5)])
+
+    mask_cube = Spectrum1D(flux=np.ones_like(spectrum1d_cube_larger.flux),
+                           spectral_axis=spectrum1d_cube_larger.spectral_axis,
+                           mask=np.ones_like(spectrum1d_cube_larger.flux))
```

Comment:
Kinda confusing for a mask cube to also have mask. What does setting mask to ones here mean?